nsd.conf.sample.in

```diff
@@ -248,6 +248,38 @@ server:
 	# tls-service-ocsp: "path/to/ocsp.pem"
 	# tls-port: 853
 
+verify:
+	# Enable zone verification. Default is no.
+	# enable: no
+
+	# Port to answer verifier queries on. Default is 5347.
+	# port: 5347
+
+	# Interfaces to bind for zone verification (default are the wildcard
+	# interfaces 0.0.0.0 and ::0). To bind to to multiple IP addresses,
```

Comment:
Default of 127.0.0.1 and ::1 is more secure.

Reply:
Actually changed to code to pass "localhost" so it'll work as expected and open both IPv4 and IPv6 sockets.